src/proof/cnf_proof.cpp

```diff
@@ -431,6 +465,31 @@ void LFSCCnfProof::printCnfProofForClause(ClauseId id,
 
   Assert( clause->size()>0 );
 
+  // If the clause contains x v ~x, it's easy!
+  //
+  // It's important to check for this case, because our other logic for
+  // recording the location of variables in the clause assumes the clause is
+  // not tautological
+  Maybe<std::pair<unsigned, unsigned>> isTrivialTaut =
+      detectTrivialTautology(*clause);
+  if (isTrivialTaut.just())
+  {
+    unsigned posIndexInClause = isTrivialTaut.value().first;
+    unsigned negIndexInClause = isTrivialTaut.value().second;
+    Trace("cnf-pf") << "; Indices " << posIndexInClause << " (+) and "
+                    << negIndexInClause << " (-) make this clause a taut"
+                    << std::endl;
+
+    std::string proofOfPos =
+        ProofManager::getLitName((*clause)[negIndexInClause], d_name);
+    std::string proofOfNeg =
+        ProofManager::getLitName((*clause)[posIndexInClause], d_name);
+    os << "(contra _ " << proofOfPos << " " << proofOfNeg << ")";
+  }
+  else
+  {
```

Comment:
The indentation of the code within those parentheses is off.

Reply:
Yeah. I'll format this file.

Reply:
Hmm, formatting the whole file is a bit excessive and makes it hard to see your changes. Any chance that you could just format your changes? [These](https://github.com/CVC4/CVC4/wiki/Developer-Guide#formatting-with-clang-format) explanations may be helpful. The easiest way is probably to do a `git reset HEAD~1` to remove your last commit, then add the non-formatting changes to a separate commit with `git add -p`, commit those changes, discard the remaining changes, then something like `git clang-format master`.

Reply:
You're right -- thanks for keeping me honest...

I'll do it right and force-push over this.